### PR TITLE
✨ Add Tour info

### LINKED ISSUE #47: FahrtenInfo erstellen
Wie in XD

### PR DESCRIPTION
**Pull request by bwnstck**
# Description
Add the Info Page for Cards

Fixes #47 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: client/src/components/Card.js [hunk 1]
```diff
@@ -108,7 +107,19 @@ const InfoContainer = styled.div`
   padding: 0rem 0 0.2rem;
 `;
 
-const Card = ({ type, ...props }) => {
+const LabelContainer = styled.div`
+  display: flex;
+  /* justify-content: space-evenly; */
```
Comment (kschmitz123) [inline client/src/components/Card.js]: delete if not needed

**Review by kschmitz123 — APPROVED** [review]
lgtm 👍

File: client/src/GlobalStyles.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { createGlobalStyle } from "styled-components";
+import { createGlobalStyle } from "styled-components/macro";
```
Comment (ncbecker) [inline client/src/GlobalStyles.js:1]: What is the reason for the change? I think the original import from "styled-components" should work for GlobalStyle?
https://styled-components.com/docs/api#createglobalstyle

**Review by ncbecker — APPROVED** [review]

File: client/src/GlobalStyles.js (hunk 2 above)
Comment (arassaria) [reply client/src/GlobalStyles.js:1]: > What is the reason for the change? I think the original import from "styled-components" should work for GlobalStyle?
> https://styled-components.com/docs/api#createglobalstyle

The difference between the import from "styled-components" to "styled-components/macro" is that with that little "macro" added, the classnames will be more clearly named. Thats how i understood the difference so far.

**Review by arassaria — APPROVED** [review]
🦄 LGTM 🚀

File: client/src/components/Todo.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styled from "styled-components";
```
Comment (SvLowman) [inline client/src/components/Todo.js]: ... from "styled-componens/macro";

**Review by SvLowman — APPROVED** [review]
Nice job 😊
